back/api/views.py

```diff
@@ -2,11 +2,28 @@
 from rest_framework.response import Response
 from rest_framework.decorators import api_view
 from rest_framework.views import APIView
-from .models import User
+from .models import *
 from rest_framework.exceptions import AuthenticationFailed
-from .serializers import UserSerializer
+from .serializers import *
 import jwt, datetime
 # Create your views here.
+def authenticated(function):
+    def wrapper(self, request, *args):
```

Comment:
You'll be interested by `functools.wraps`

https://docs.python.org/3/library/functools.html#functools.wraps

Reply:
Thanks for the tip